Numerics/CustomModAdd/CustomModAdd.qs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Numerics.Samples
```

Comment:
In general, Q# and C# styles differ on where to put braces, such that I'd suggest braces be on the same line for Q# code.